packages/config/readme.md

```diff
@@ -1,4 +1,126 @@
+| `basePath` | `String` | Root path of your application on the server. This is inserted as an HTTP path prefix between `host` and `locations`. For example, if you set e.g. `"basePath": "my-custom-basepath"` and `"location": ["/profile"]`, then your application would be available at `http://0.0.0.0:8080/my-custom-basepath/my-custom-location/` |
+| `assetPath` | `String` | HTTP root path of your asset files on the server. It's not dependent on `basePath`, you can define totally separate directories |
+| `browsers` | `String` | [Browserslist](https://www.npmjs.com/package/browserslist) query. Defaults to `"> 1%, last 2 versions, Firefox ESR"` (which is also the Browserlist default query). It is used for Babel and PostCSS |
+| `moduleDirs` | `Array<string>` | One or more directories that Hops should use for module path resolution. For example, you can specify a directory that holds a yet unpublished Node.js package. It works similar to the [resolve.modulesDirectories](http://webpack.github.io/docs/configuration.html#resolve-modulesdirectories) field in Webpack, i.e. if you specify `moduleDirs: ["my-module-dir"]`, it will look in `./my-module-dir`, `../my-module-dir`, `../../my-module-dir` and so on |
+| `appDir` | `String` | Directory of your app. This way, you can put your app to a place other than project root. Defaults to `"."` |
+| `buildDir` | `String` | Directory where the build artifacts (i.e. your application/asset files) will be saved to. Defaults to `"build"`. Hops always removes the specified directory before starting a new build |
+| `cacheDir` | `String` | Directory where internally used artifacts not meant to be served will be output (e.g. the `manifest.json`/`manifest.js` files generated by Webpack). Defaults to `"node_modules/.cache/hops"` |
+| `buildConfig` | `String` | Path to your Webpack build configuration file. Defaults to Defaults to `./node_modules/hops-config/configs/build.js` |
+| `developConfig` | `String` | Path to your Webpack development configuration file. Defaults to `./node_modules/hops-config/configs/develop.js` |
+| `nodeConfig` | `String` | Path to your Webpack node/server-side rendering configuration file. Defaults to `./node_modules/hops-config/configs/node.js` |
+| `manifest` | `String` | The application's `manifest.js` as a string. Defaults to a getter function that returns the content of the automatically generated `manifest.js` |
+| `assets` | `Object literal` | The applications's `.js` and `.css` assets. Defaults to a getter function that returns the assets contained in the automatically generated `manifest.json` |
+| `extends` | `String` | Path to a baseline configuration (could be for example a node module or a project-specific file), allowing you join the current configuration with that baseline |
+
+hops-config normalizes all specified directory and path strings. That means the following statements yield the same result:
```

Comment:
> I'm afraid we're mixing up two separate features here: normalization happens for some HTTP path related values (`locations`, `basePath`, `assetPath`) and is something I probably wouldn't even mention here.

Actually I had that correctly in a previous version already with `basePath`. But anyways, I learned that such sentence doesn't tell the complete story, because the behavior differs for each of these keys. So let's omit it.

> On the other hand, everything matching `/(config|file|dir)s?$/i` will be treated as a filesystem path (or array thereof) and resolved relative to app root. This only applies if the respective value is not an absolute path already.

Okay, let's add that to the page. One thing, though, when you say "relative to app root": Maybe we should say "relative to project root" in order to avoid people mixing up app root with appDir?